plugins/modules/user.py

```diff
@@ -189,6 +207,15 @@ def main():
     client = arguments.get_sensu_client(module.params['auth'])
     path = utils.build_core_v2_path(None, 'users', module.params['name'])
 
+    try:
+        if not HAS_BCRYPT and client.version >= "5.21.0":
+            module.fail_json(
+                msg=missing_required_lib('bcrypt'),
+                exception=BCRYPT_IMPORT_ERROR,
+            )
+    except errors.SensuError as e:
+        module.fail_json(msg=str(e))
```

Comment:
Just wondering if this part deserves a unit test. Maybe not so much for the `except` to cover errors stemming from `client.version`, but for the bcrypt dependency part - at least a test to ensure failure when we're missing the required library with 5.21.0 or newer? I would love to see something along the lines of `test_failure_on_missing_bcrypt_5_21_0_or_newer`.